docs-conceptual/azurermps-5.7.0/install-azurerm-ps.md

```diff
@@ -38,161 +40,137 @@ PowerShellGet 1.6.0   C:\Program Files\WindowsPowerShell\Modules\PowerShellGet\1
 
-## Step 3: Load the AzureRM module
-Once the module is installed, you need to load the module into your PowerShell session. You should
-do this in a normal (non-elevated) PowerShell session. Modules are loaded using the `Import-Module`
-cmdlet, as follows:
+To start working with the Azure PowerShell module, you need to load it into your current PowerShell session
+with the [Import-Module](/powershell/module/Microsoft.PowerShell.Core/Import-Module) cmdlet, and then log in
+with your Azure credentials.
 
 ```powershell
-Import-Module -Name AzureRM
+# Import the module into the PowerShell session
+Import-Module AzureRM
+# Connect to Azure with an interactive dialog for login
+Connect-AzureRmAccount
 ```
 
-## Next Steps
-
-For more information about using Azure PowerShell, see the following articles:
-
-* [Get started with Azure PowerShell](get-started-azureps.md)
+To learn about persisting your sign in credentials across sessions, see [Persist user logins across PowerShell sessions](context-persistence.md).
 
-## Reporting issues and feedback
+## Uninstall the Azure PowerShell module
 
-If you encounter any bugs with the tool, file an issue in the
-[Issues](https://github.com/Azure/azure-powershell/issues) section of our GitHub repo. To provide
-feedback from the command line, use the `Send-Feedback` cmdlet.
+To uninstall the Azure PowerShell module, use the [Uninstall-Module](/powershell/module/powershellget/uninstall-module) cmdlet. To uninstall 
+the most recent version of the Azure PowerShell on your system:
 
-## Frequently asked questions
+```powershell
+Uninstall-Module -Name AzureRM
+```
 
-### How to get PowerShellGet
+You can also delete a single version:
 
-|Scenario|Install instructions|
-|---|---|
-|Windows 10<br/>Windows Server 2016|Built into Windows Management Framework (WMF) 5.0 included in the OS|
-|I want to upgrade to PowerShell 5|<ol><li>[Install the latest version of WMF](https://www.microsoft.com/en-us/download/details.aspx?id=54616)</li><li>Run the following command:<br/>```Install-Module PowerShellGet -Force```</li></ol>|
-|I am running on a version of Windows with PowerShell 3 or PowerShell 4|<ol><il>[Get the PackageManagement modules](http://go.microsoft.com/fwlink/?LinkID=746217)</il><li>Run the following command:<br/>```Install-Module PowerShellGet -Force```</li></ol>|
-
-<a id="helpmechoose"></a>
-### Checking the version of Azure PowerShell
+```powershell
+Uninstall-Module -Name AzureRM -RequiredVersion 6.1.0
+```
 
-Although we encourage you to upgrade to the latest version as early as possible, several versions
-of Azure PowerShell are supported. To determine the version of Azure PowerShell you have installed,
-run `Get-Module AzureRM` from your command line.
+Or completely remove Azure PowerShell from your system.
 
 ```powershell
-Get-Module AzureRM -ListAvailable | Select-Object -Property Name,Version,Path
+Uninstall-Module -Name AzureRM -AllVersions
```

Comment:
Actually, that command only uninstalls the AzureRM module. It does not uninstall any of the submodules. You have to run Uninstall-Module on each AzureRM.* module. This is a limitation of PowerShellGet. I am not sure we want this in the install article. I suggest a separate uninstall article. Or strike this from this article.

Reply:
Will strike from this article - can you submit a preliminary "Uninstall" article that I can edit in? Either as a branch off your fork, or as a PR to this repo. I think that I can edit either with my level of permissions.